src/Files.App/Actions/Navigation/OpenDirectoryInNewPaneAction.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal class OpenDirectoryInNewPaneAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		public string Label
+			=> "OpenDirectoryInNewPane".GetLocalizedResource();
+
+		public string Description
+			=> "OpenDirectoryInNewPaneDescription".GetLocalizedResource();
+
+		public bool IsExecutable
+			=> context.HasSelection;
```

Comment:
On second thought, we should limit this to having a single folder selected.

Reply:
@ferrariofilippo What conditions you think am I supposed to add? I have no idea, following two ones as well.

Reply:
I think, as @yaira2 pointed out, that we should check if:
- anything is selected
- just one item is selected (we could change this once we refactor `PaneHolderPage` so that it can holds > 2 panes)
- that item is a folder

Reply:
Fixed. Up to 5.